pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -26,6 +26,12 @@ self: super: {
   # successfully with recent versions of the compiler).
   bin-package-db = null;
 
+  # release is taking long
```

Comment:
Could you create an issue upstream asking for a release (or link to an issue if it already exists), and then put that issue here in a comment.

This is so that we know when this override can be removed.

Reply:
https://github.com/jwiegley/c2hsc/issues/41